### PR TITLE
Network and Some Random mappings

### PR DESCRIPTION
**Pull request by magik6k**

### CODE REVIEW

File: mappings/net/minecraft/client/network/handler/NetworkLoginHandlerClient.mapping [hunk 1]
```diff
@@ -3,6 +3,7 @@ CLASS none/bme net/minecraft/client/network/handler/NetworkLoginHandlerClient
 	FIELD b game Lnone/bdr;
 	FIELD c parentGui Lnone/bgn;
 	FIELD d connection Lnone/eq;
+	FIELD e profile Lcom/mojang/authlib/GameProfile;
```
Comment (copygirl) [inline mappings/net/minecraft/client/network/handler/NetworkLoginHandlerClient.mapping]: Maybe `playerProfile` ?

File: mappings/net/minecraft/network/NetworkEncryptionUtils.mapping [hunk 2]
```diff
@@ -1,4 +1,24 @@
 CLASS none/og net/minecraft/network/NetworkEncryptionUtils
 	FIELD a LOGGER Lorg/apache/logging/log4j/Logger;
-	METHOD a ()Ljavax/crypto/SecretKey;
+	METHOD a generateKey ()Ljavax/crypto/SecretKey;
+	METHOD a crypt (ILjava/lang/String;Ljava/security/Key;)Ljavax/crypto/Cipher;
+		ARG 0 opmode
```
Comment (copygirl) [inline mappings/net/minecraft/network/NetworkEncryptionUtils.mapping]: `opMode` ? (+ other occurrences)

File: mappings/net/minecraft/sortme/VillageProperties.mapping [hunk 3]
```diff
@@ -0,0 +1,42 @@
+CLASS none/wg net/minecraft/sortme/VillageProperties
+	FIELD a world Lnone/aiw;
+	FIELD b doors Ljava/util/List;
+	FIELD c doorPositionsAggregate Lnone/cn;
+	FIELD d center Lnone/cn;
+	FIELD e radius I
+	FIELD f stable I
+	FIELD g tick I
+	FIELD h populationSize I
+	FIELD i mtick I
```
Comment (copygirl) [inline mappings/net/minecraft/sortme/VillageProperties.mapping]: `mtick` ? Very non-descriptive to me.

Comment (magik6k) [reply mappings/net/minecraft/sortme/VillageProperties.mapping]: http://minecraft.gamepedia.com/Villages.dat_format

Comment (copygirl) [reply mappings/net/minecraft/sortme/VillageProperties.mapping]: `lastVillagerDeath(Time)` ? Should I take a look at it myself?

File: mappings/net/minecraft/network/ClientConnection.mapping [hunk 4]
```diff
@@ -1,34 +1,68 @@
...
 	FIELD m packetListener Lnone/ev;
 	FIELD n disconnectReason Lnone/fa;
+	FIELD o encrypted Z
+	FIELD p disconnected Z
 	METHOD <init> (Lnone/fm;)V
 		ARG 0 side
 	METHOD a setMinCompressedSize (I)V
 		ARG 0 value
+	METHOD a (Lio/netty/channel/ChannelHandlerContext;Lnone/fl;)V
+		ARG 0 ctx
```
Comment (maxpowa) [inline mappings/net/minecraft/network/ClientConnection.mapping]: Why `ctx` when the existing mappings here refer to this param type as `context` ?

Related: All `ctx` names in this PR.

File: mappings/net/minecraft/sortme/VillageProperties.mapping (hunk 3 above)
Comment (asiekierka) [reply mappings/net/minecraft/sortme/VillageProperties.mapping]: Please do. mtick is not descriptive, and we shouldn't just copy NBT names.

**Pull request merged**